### DISCUSSION TITLE
Disable BB logo on self-hosted plan

### DISCUSSION THREAD

**Discussion by Klience** [Budibase/budibase#8366] [Help] [answered]
Does the self-hosted version of Budibase have logo always enabled?

If yes, is it possible to disable it?

EDIT: Is the self-hosted plan limited to 4,000 rows per app too?

**Comment by melohagan** [comment, marked as answer]
This info can be found on the pricing page! https://budibase.com/pricing/

**Reply by Klience** [reply]
Hi @melohagan,

Sorry I can't seem to find it. The pricing page for the self-hosted version only mentions unlimited apps and users.

It doesn't say anything about the number of rows for the open-source plan.

**Reply by melohagan** [reply]
Ah, self-hosted allows for unlimited rows per app because it's your own infrastructure.

**Reply by Klience** [reply]
Thank you! Appreciate the quick response.
